easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2022a.eb

```diff
@@ -0,0 +1,48 @@
+easyblock = 'PythonBundle'
+
+name = 'kb-python'
```

Comment:
We try to stick to the same naming as the project refers to itself. I would suggest `kallisto-bustools`

Reply:
Does this than have an impact on this easyconfig [kb-python-0.27.3-foss-2021b.eb](https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/k/kb-python/kb-python-0.27.3-foss-2021b.eb)? Will the software than not be available under multiple names in easybuild?